requirements.txt

```diff
@@ -15,3 +15,4 @@ scikit-image==0.19.3
 scipy==1.9.3  
 xarray==2023.1.0
 eofs==1.4.0
+sphinx-design==0.3.0
```

Comment:
```suggestion
```

Reply:
Please remove "sphinx-design==0.3.0" from the requirements.txt file.  It only needs to be in the docs/requirements.txt as it is only a requirement to build the documentation and not a requirement for using METplotpy.